--- a/src/libslic3r/CMakeLists.txt
+++ b/src/libslic3r/CMakeLists.txt

@@ -480,6 +480,8 @@
 endif ()
 
 add_library(libslic3r STATIC ${lisbslic3r_sources}
+set(SKIP_LIBSLIC3R_CGAL ON)
+
     "${CMAKE_CURRENT_BINARY_DIR}/libslic3r_version.h"
     ${OpenVDBUtils_SOURCES})
 source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${lisbslic3r_sources})
@@ -495,12 +497,9 @@
 find_package(OpenCV REQUIRED core)
 cmake_policy(POP)
 
+if (NOT SKIP_LIBSLIC3R_CGAL)
 add_library(libslic3r_cgal STATIC
-    CutSurface.hpp CutSurface.cpp
-    IntersectionPoints.hpp IntersectionPoints.cpp
-    MeshBoolean.hpp MeshBoolean.cpp
-    TryCatchSignal.hpp TryCatchSignal.cpp
-    Triangulation.hpp Triangulation.cpp)
+
 target_include_directories(libslic3r_cgal PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 
 # Reset compile options of libslic3r_cgal. Despite it being linked privately, CGAL options
@@ -526,7 +525,9 @@
     set_property(TARGET libslic3r_cgal PROPERTY CXX_EXTENSIONS OFF)
 endif()
 
+if (NOT SKIP_LIBSLIC3R_CGAL)
 target_link_libraries(libslic3r_cgal PRIVATE ${_cgal_tgt} libigl mcut)
+endif()
 
 if (MSVC AND "${CMAKE_SIZEOF_VOID_P}" STREQUAL "4") # 32 bit MSVC workaround
     target_compile_definitions(libslic3r_cgal PRIVATE CGAL_DO_NOT_USE_MPZF)
