--- CMakeLists.txt.orig	2025-12-25 20:39:22.057776646 +0100
+++ CMakeLists.txt	2025-12-25 20:40:38.401764841 +0100
@@ -495,6 +495,8 @@
 find_package(OpenCV REQUIRED core)
 cmake_policy(POP)
 
+set(SKIP_LIBSLIC3R_CGAL ON)
+if (NOT SKIP_LIBSLIC3R_CGAL)
 add_library(libslic3r_cgal STATIC
     CutSurface.hpp CutSurface.cpp
     IntersectionPoints.hpp IntersectionPoints.cpp
@@ -502,6 +504,7 @@
     TryCatchSignal.hpp TryCatchSignal.cpp
     Triangulation.hpp Triangulation.cpp)
 target_include_directories(libslic3r_cgal PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+endif()
 
 # Reset compile options of libslic3r_cgal. Despite it being linked privately, CGAL options
 # (-frounding-math) still propagate to dependent libs which is not desired.
